src/box/txn.c

```diff
@@ -296,6 +297,7 @@ txn_add_redo(struct txn *txn, struct txn_stmt *stmt, struct request *request)
 	 */
 	struct space *space = stmt->space;
 	row->group_id = space != NULL ? space_group_id(space) : 0;
+	wal_ext_space_process(space, stmt, request);
```

Comment:
Better call this function `wal_ext_process_request`, because it processes a request, not a space.